realtime.go

```diff
@@ -99,16 +99,37 @@ type StopTimeEvent struct {
 }
 
 type VehicleID struct {
-	ID           string
-	Label        string
-	LicencePlate string
+	ID           *string
+	Label        *string
+	LicencePlate *string
 }
 
+type Position = gtfsrt.Position
+type CurrentStatus = gtfsrt.VehiclePosition_VehicleStopStatus
+type CongestionLevel = gtfsrt.VehiclePosition_CongestionLevel
+type OccupancyStatus = gtfsrt.VehiclePosition_OccupancyStatus
+
 type Vehicle struct {
 	ID *VehicleID
 
 	Trip *Trip
 
+	Position *Position
+
+	CurrentStopSequence *uint32
+
+	StopID *string
+
+	CurrentStatus *CurrentStatus
+
+	Timestamp *time.Time
+
+	CongestionLevel *CongestionLevel
+
+	OccupancyStatus *OccupancyStatus
```

Comment:
and also make this non-nil, with a default of `NO_DATA_AVAILABLE`? If you think it would be good.

Reply:
I didn't see a good default (something that clearly means "value not present") for `OccupancyStatus` or `CurrentStatus` enums unfortunately, unless I am missing something. Or was the suggestion here to create new enums?

Reply:
My (hacky!) suggestion was to piggyback on the existing enum value `NO_DATA_AVAILABLE`. But maybe let's again punt it. In the future we may add a non-proto enum, and then we can have a specific value that represents "the feed did not set this field".